src/pages/PointsPage/point_display.js

```diff
@@ -49,6 +49,7 @@ class PointDisplay extends React.Component {
           <div>
             <Grid container spacing={2}>
               {points.map(event => {
+                console.log(event.date);

```

Comment:
hehe

Reply:
lolol rip :)) I ended up figuring out its type thru something else, I'll fix this.